pkg/workloads/console/controller.go

```diff
@@ -423,11 +424,30 @@ func (r *reconciler) buildJob(template *workloadsv1alpha1.ConsoleTemplate) *batc
 	// 1 pod per job.
 	backoffLimit := int32(0)
 
+	// jobName: Add console postfix and truncate name no longer than 49
+	// characters (63-14) (14 characters '-console-4mh4z')
+	jobName := fmt.Sprintf("%s-%s", stringTruncate(r.name.Name, 49), "console")
+
+	// Merged labels from the console template and console
+	// In case of conflits second label set wins
```

Comment:
Typo: `conflicts`

I think that this comment might be better if we explained *why* we're giving the labels from the template precedence; i.e.
> The labels on the console can be user-defined, so we do not want to allow a user to create a console with a label that implies that it's for an application different to the console.